src/context/db.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext, useContext, useState } from "react";
+import { RxDatabase } from "rxdb";
+
+export const DbContext = createContext({});
+
+export const useDbContext = (): { db: RxDatabase; setDb: Function } =>
+  useContext(DbContext) as any;
+
+interface DbContextProviderProps {
+  children: object;
+}
+
+export const DbContextProvider = ({ children }: DbContextProviderProps) => {
```

Comment:
We can do `export const DbContextProvider: React.FC = ({ children }) => {`, which will automatically include the `children` prop

Reply:
With React.FC it's still showing `  16:47  error  'children' is missing in props validation  react/prop-types`

Reply:
Shall we add `"react/prop-types": 0` to our eslint? We used it here too: https://github.com/Open-Attestation/react-component-template/blob/master/package.json#L138